Core/Utilities/InstallTools.py

```diff
@@ -616,11 +618,7 @@ def addDefaultOptionsToCS( gConfig, componentType, systemName,
   if not compInstance:
     return S_ERROR( '%s not defined in %s' % ( instanceOption, cfgFile ) )
 
-  sectionName = "Agents"
-  if componentType == 'service':
-    sectionName = "Services"
-  elif componentType == 'executor':
-    sectionName = "Executors"
+  sectionName = _getSectionName ( componentType )[ 'Value' ]
```

Comment:
The _getSectionName function returns S_OK structure. In this case the reault['OK'] must be checked and not assumed to be True. This function indeed can never return S_ERROR. If this is a trivial function that can never fail, just return a value without the S_OK/S_ERROR structure.